docker/docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
 
     sqe-http:
         container_name: SQE_HTTP_API
-        image: qumranica/sqe-http-api:0.4.5-pre6
+        image: qumranica/sqe-http-api:0.5.0-pre12
```

Comment:
If you want, you can change this now to 0.5.0, as you mentioned at the top of the PR.